sdk/resourcemanager/Proto.Client/authorization/README.MD

```diff
@@ -0,0 +1,3 @@
+# Proto Authorization
```

Comment:
What is the ultimate goal for this Proto.Client? Is it is going to become a library we ship? Is it just going to be for testing or is it going to be a sample? If it is going to be a test or sample we should put it under the tests or samples folder. I

Reply:
This is temporary until the autogen work is done it allows us to test our arm core in a working end to end state